### DISCUSSION TITLE
Automatic Unrar setup / command line inside transmission settings

### DISCUSSION THREAD

**Discussion by C0nw0nk** [haugene/docker-transmission-openvpn#2234] [Q&A]
So i read that in haugene docker-transmission package it comes with unrar abilities.

![image](https://user-images.githubusercontent.com/12204587/161385161-d1a503ce-4379-445d-96b0-87d26a4ede44.png)

I have setup everything but files that download as compressed archives be it zip / rar / 7z etc don't automatic extract.

What is the command for transmission to do this upon completion of a download ?

**Comment by pkishino** [comment]
see #701 #75 #1283 should give you some ideas

**Reply by C0nw0nk** [reply]
Thanks yeah i read through those prior to posting while the script would work it is a bit hacky and not exactly the most elegant way of doing it.

For those using sonarr and radarr etc with transmission you will get a error on the que in the program until it extracts.

**Comment by C0nw0nk** [comment]
The way I am currently doing it is like this.

I have setup another docker container and I have installed this docker container to unpack compressed archives in the `completed` folder of transmission.

https://registry.hub.docker.com/r/golift/unpackerr

Its not using the built in method but there is no documentation or examples so this is my soloution for now.

In my opinion it works better with sonarr radarr lidarr etc like this because it extracts and deletes from the ques and manages it all automatically and correctly.
